mbed-trace/mbed_trace.h

```diff
@@ -63,18 +63,18 @@ extern "C" {
 #endif
 
 #ifndef YOTTA_CFG_MBED_TRACE_FEA_IPV6
-#define YOTTA_CFG_MBED_TRACE_FEA_IPV6 1
+#define YOTTA_CFG_MBED_TRACE_FEA_IPV6 0
```

Comment:
Oh and one more thing... If you are including nanostack-libservice anyway there's probably no need to disable IPv6 tracing features. They will work and may be helpful if any application wants to trace IPv6 addresses. Plus if nanostack is used for networking, it may "need" them if built with tracing enabled.